sites/public/src/components/listings/Pagination.tsx

```diff
@@ -53,7 +58,7 @@ export function Pagination(props: PaginationProps) {
     const styleType = isCurrent ? AppearanceStyleType.primary : null
 
     pageButtons.push(
-      <Button styleType={styleType} onClick={onClick}>
+      <Button styleType={styleType} onClick={onClick} size={AppearanceSizeType.small}>
```

Comment:
This, combined with some CSS changes, makes the pagination buttons smaller and rounder

<img width="605" alt="Screenshot 2024-11-12 at 8 17 41 PM" src="https://github.com/user-attachments/assets/d0612108-58d3-4816-b91b-b7000ebc132d">
<img width="604" alt="Screenshot 2024-11-12 at 8 17 47 PM" src="https://github.com/user-attachments/assets/55a69c17-febd-4401-a864-fdcf1f3eb905">